Modules/Tags.Application/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Backbone.Modules.Tags.Application.Tags.Queries.ListTags;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Backbone.Modules.Tags.Application.Extensions;
+
+public static class IServiceCollectionExtensions
+{
+    public static void AddApplication(this IServiceCollection services)
+    {
+        services.AddMediatR(c => c
+            .RegisterServicesFromAssemblyContaining<ListTagsQuery>()
+        );
+    }
+
+    public static void AddTags(this IServiceCollection services, ApplicationOptions applicationOptions)
+    {
+        services.AddSingleton(_ => new TagProvider(applicationOptions.Attributes));
```

Comment:
You should rather use a `(I)TagsRepository` to match the existing pattern. The only difference to the other repositories would be that it reads from configuration instead of from the database.
This way, we don't have to change any existing code when we want to read the tags from the database one day